lib/administrate/base_dashboard.rb

```diff
@@ -18,6 +18,18 @@ def attribute_types
       self.class::ATTRIBUTE_TYPES
     end
 
+    def attribute_type_for(attribute_name)
+      attribute_types.fetch(attribute_name) do
+        fail "Attribute #{attribute_name} could not be found in #{self.class}::ATTRIBUTE_TYPES"
```

Comment:
oh, sorry - I think it's a good idea to extract the message into its own method regardless. I meant that if hound comments again, we can ignore that.